src/components/pages/errorPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PageWrapper from 'components/shared/layouts/pageWrapper'
+
+import styles from './errorPage.scss'
+
+type Props = {
+    error: Error | null
+}
+
+const ErrorPage = ({error}: Props) => (
+    <PageWrapper>
+        <div className={styles.errorBackground}>
+            <div className={styles.errorContainer}>
+                <h1 className={styles.error}>Error</h1>
+                <p className={styles.errorMessage}>{error?.message}</p>
```

Comment:
Ah perfect. This is what I was talking about. in this like do 
```suggestion
                <p className={styles.errorMessage}>{error?.message || defaultErrorMessage}</p>
```

then add a `defaultErrorMessage` at the top of this component